### PR TITLE
CONCF-743 | remove data-crop when removing image in desktop CC tool

### PR DESCRIPTION
**Pull request by Warkot**
Also fixed some lint errors.

/cc @vforge

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js [hunk 1]
```diff
@@ -445,7 +457,7 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 				on('click', '.photo-remove', function () {
 					var $this = $(this),
 						$line = $this.parent(),
-						$image = $line.find('.image').removeAttr('data-id');
+						$image = $line.find('.image').removeAttr('data-id').removeAttr('data-crop');
```
Comment (Warkot) [inline extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js]: @vforge here's the core change.

Comment (vforge) [inline extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js]: :+1:

**Comment by vforge** [comment]
lgtm

**Pull request merged**